.github/workflows/testing.yml

```diff
@@ -0,0 +1,48 @@
+name: Testing
+# in this file we describe the action to execute for the CI in the dev pipeline
+
+# based on default laravel workflow
+# https://github.com/actions/starter-workflows/blob/master/ci/laravel.yml
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  laravel-tests:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout to current branch
+      run: git fetch && git checkout ${{ github.head_ref }}
+    - name: Copy .env
+      run: php -r "file_exists('.env') || copy('.env.example', '.env');"
+      working-directory: ./src
+    - name: Install Dependencies
+      run: composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
+      working-directory: ./src
+    - name: Generate key
+      run: php artisan key:generate
+      working-directory: ./src
+    - name: Directory Permissions
+      run: chmod -R 777 storage bootstrap/cache
+      working-directory: ./src
+    - name: Create Database
+      run: |
+        mkdir -p database
+        touch database/database.sqlite
```

Comment:
Same comment as before

Reply:
Fixed in 2b736399840ba074fc05ec056276a0853684d7ab ae011bb9f04794b06fe1a6e4a5a02cff7a0a2b1a